cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -74,7 +74,7 @@
     "name": "kube-controller-manager",
     "image": "{{pillar['kube_docker_registry']}}/kube-controller-manager:{{pillar['kube-controller-manager_docker_tag']}}",
     "resources": {
-      "limits": {
```

Comment:
We default request from limit if limit is provided but request is not. By changing this to limit, we maintian the behavior we had before enabling cpu-cfs-quota. As for why it was set to limits in the first place, I doubt that there was ever an intention to hardcap these.